Directory.Build.props

```diff
@@ -18,8 +18,8 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<None Include="$(MSBuildStartupDirectory)\images\nitro-devs-logo.png" Pack="true" Visible="false" PackagePath="" />
-		<None Include="$(MSBuildStartupDirectory)\README.md" Pack="true" PackagePath=""/>
+		<None Include="$(MSBuildThisFileDirectory)images\nitro-devs-logo.png" Pack="true" Visible="false" PackagePath="" />
```

Comment:
I don't believe this needed to change

Reply:
The variable `MSBuildStartupDirectory` do not work correctly on IDE, because of that I've changed it. The variable `MSBuildThisFileDirectory` works on any scenario, as it is always relative to current file.